### DISCUSSION TITLE
Github Discussions are now connected to ESA-NASA-MAAP Slack

### DISCUSSION THREAD

**Discussion by wildintellect** [MAAP-Project/Community#764] [Announcements]
Users are now advised to open a [discussion](https://github.com/orgs/MAAP-Project/discussions) question to help record questions and answers for the whole MAAP community. 

All questions opened on Github https://github.com/orgs/MAAP-Project/discussions will be posted to Slack to allow for real time problem solving. When a final solution is reached, it's requested that it be posted back to the Discussion on Github for future search-ability and reference. The entire conversation can also happen on Github as responses will also be added to Slack as threads on the post. In the event that a Discussion leads to the discovery of a bug or new feature request the Discussion will be transferred to an [Issue](https://github.com/MAAP-Project/Community/issues) for the developers.

This is an early experiment in categories so please feel free to make suggestions to for improvement.
| Github | ESA-NASA-MAAP Slack |
| - | - |
| [Announcements](https://github.com/orgs/MAAP-Project/discussions/categories/announcements) | [Announcements](https://esa-nasa-maap.slack.com/archives/C05JGHRCM6E) |
| [Data](https://github.com/orgs/MAAP-Project/discussions/categories/data) | [Data](https://esa-nasa-maap.slack.com/archives/C05J4SR9021) | 
| [Platform](https://github.com/orgs/MAAP-Project/discussions/categories/platform) | [Platform](https://esa-nasa-maap.slack.com/archives/CLQB3B620) | 
| [Issues](https://github.com/MAAP-Project/Community/issues) | [Issues](https://esa-nasa-maap.slack.com/archives/C05EQV8CK9R) |
| Science: [Q&A](https://github.com/orgs/MAAP-Project/discussions/categories/q-a), [Show and Tell](https://github.com/orgs/MAAP-Project/discussions/categories/show-tell) | [UWG](https://esa-nasa-maap.slack.com/archives/C01308ZHELF) |

This is what the Discussions sections and categories are currently as listed in the table above.
![image](https://github.com/MAAP-Project/Community/assets/306179/9006349e-98d3-428f-95e4-e5b2de30c209)

**Comment by brianmfreitag** [comment]
@wildintellect - slight modifications below. The links in the original ticket are preserved in the email, but otherwise take a look and let me know if everything looks okay before i send.

Hello again MAAP Users,

This email serves as an announcement of a new form of communication we are testing for MAAP users to pose questions to the dev teams and the MAAP community at-large. Users are now advised to open a discussion question to help record questions and answers for the whole MAAP community.
All questions opened on Github https://github.com/orgs/MAAP-Project/discussions will be posted to Slack to allow for real time problem solving. When a final solution is reached, it's requested that it be posted back to the Discussion on Github for future search-ability and reference. The entire conversation can also happen on Github as responses will also be added to Slack as threads on the post. In the event that a Discussion leads to the discovery of a bug or new feature request the Discussion will be transferred to an Issue for the developers.

Below shows a mapping of where you can find the discussion sections and categories as defined in GitHub within the ESA-NASA-MAAP Slack workspace as separate channels (e.g. #announcements, #data, etc.). These categories are considered experimental so please feel free to make suggestions to for improvement.

Github | ESA-NASA-MAAP Slack
-- | --
Announcements | Announcements
Data | Data
Platform | Platform
Issues | Issues
Science: Q&A, Show and Tell | UWG

Below is similar information but shows how the discussions sections and categories are organized in GitHub.

![](https://user-images.githubusercontent.com/306179/255961997-9006349e-98d3-428f-95e4-e5b2de30c209.png)

Please let us know if there are any questions/concerns.

**Comment by brianmfreitag** [comment]
proposed subject line: New Feature: GitHub Discussions for community engagement with developers
